src/mpq.rs

```diff
@@ -70,17 +71,9 @@ impl Mpq {
         &mut self.mpq
     }
 
-    pub fn new() -> Mpq {
```

Comment:
We can’t go around deleting parts of the public API just because Clippy suggests it…

Reply:
Oh, very good point. My bad.

Reply:
Will fix and re-spin tomorrow. Thanks for looking at this.